index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css">
+
+    <title>Setup and mobile version skeleton</title>
+</head>
+<body>
+  <nav>
+    <ul>
+      <li id="logo"><a href="#"><img src="images/John-Doe.png" alt="Website logo"></a></li>
+      <li id ="menu"><a href="#"><img src="images/Icons/Menu.png" alt="Website menu"></a></li>
+    </ul>
+  </nav>
+<header>
+
+    <h1>Hey There.
+  I’m Alex</h1>
+    <h3>I’am a Software Developer</h3>
+    <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
+  don’t hestiate to contact me.</p>
+
+
+<div id= "social-links">
+  <ul >
+    <li><a href="#"><img src="images/Icons/GitHub.png" alt=""></a></li>
+    <li><a href="#"><img src="images/Icons/Linkedin.png" alt=""></a></li>
+    <li><a href="#"><img src="images/Icons/Angelist.png" alt=""></a></li>
+    <li><a href="#"><img src="images/Icons/Twitter.png" alt=""></a></li>
+    <li><a href="#"><img src="images/Icons/Medium.png" alt=""></a></li>
+  </ul>
+
+  <img id="illustration" src="images/illustration-header-mobile.png" alt="the illustration image">
+</div>
+
+</header>
```

Comment:
- [ ] The `<header>` element is not for sectioning content and therefore does not introduce a new section in the [outline](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements). Please change the `<header>` tag here to a `<section>` tag.